src/client/components/GraphScene.tsx

```diff
@@ -133,39 +129,24 @@ export const dataSelecter = (rawData: any, provider: string): any => {
 
 export default function GraphScene(props: GraphSceneProps): JSX.Element {
   const [graphData, setGraphData] = useState([] as any)
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const location = useLocation<Location>()
```

Comment:
May want to double-check this. Is `location` not used? If not, is line 133 needed?

Reply:
For whatever reason, it doesn't recognize that location is being used.  If I leave it in I can get the page url, but it triggers a linting error.  When I remove it I am no longer able to get the page url, but the error goes away.